pkg/utils/kubernetes/resources/jobs.go

```diff
@@ -75,6 +75,7 @@ func newWebhookCertificateConfigJobCommon(namespace, serviceAccountName string,
 			Namespace: namespace,
 		},
 		Spec: batchv1.JobSpec{
+			TTLSecondsAfterFinished: lo.ToPtr(int32(600)),
```

Comment:
Let's hold off with this one for now as this will mask the underlying problem: the jobs (with the failing `Pods` will get deleted and there will be no sign of them and no sign of failure whereas the webhook - and related resources - won't get created).

The TTL seems to be counted after the job is considered completed or failed.


```
status:
  active: 1
  ready: 0
  startTime: "2024-06-19T15:03:26Z"
  terminating: 0
  uncountedTerminatedPods: {}
```

...

```
status:
  conditions:
  - lastProbeTime: "2024-06-19T15:04:12Z"
    lastTransitionTime: "2024-06-19T15:04:12Z"
    message: Job has reached the specified backoff limit
    reason: BackoffLimitExceeded
    status: "True"
    type: Failed
  failed: 1
  ready: 0
  startTime: "2024-06-19T15:03:26Z"
  terminating: 0
  uncountedTerminatedPods: {}
```